src/core/numdiff/action.cpp

```diff
@@ -1,128 +1,9 @@
 ///////////////////////////////////////////////////////////////////////////////
 // BSD 3-Clause License
 //
-// Copyright (C) 2018-2020, LAAS-CNRS, University of Edinburgh, New York University,
-// Max Planck Gesellschaft
+// Copyright (C) 2018-2020, LAAS-CNRS, University of Edinburgh
```

Comment:
Add back the NYU and MPI for all numdiff files.
All these files were done by Maximilien.

Reply:
Yes, this was because of copy pasting. I'll fix this.